src/Extractor/ContentExtractor.php

```diff
@@ -778,6 +792,36 @@ private function removeElements($elems = false, $logMessage = null)
         }
     }
 
+    /**
+     * Wrap elements with provided tag.
+     *
+     * @param \DOMNodeList|false $elems
+     * @param string             $tag
+     * @param string             $logMessage
+     */
+    private function wrapElements($elems = false, $tag = 'div', $logMessage = null)
+    {
+        if (false === $elems || false === $this->hasElements($elems)) {
+            return;
+        }
+
+        if (null !== $logMessage) {
+            $this->logger->info($logMessage, ['length' => $elems->length]);
+        }
+
+        $a = iterator_to_array($elems);
+        foreach ($a as $item) {
+            if (null !== $item && null !== $item->parentNode) {
+                if ($item instanceof \DOMElement) {
+                    $newNode = $item->ownerDocument->createElement($tag);
+                    $newNode->setInnerHtml($item->ownerDocument->saveXML($item));
+
+                    $item->parentNode->replaceChild($newNode, $item);
+                }
+            }
```

Comment:
You can combine these ifs
```suggestion
            if (null !== $item && null !== $item->parentNode && $item instanceof \DOMElement) {
                $newNode = $item->ownerDocument->createElement($tag);
                $newNode->setInnerHtml($item->ownerDocument->saveXML($item));

                $item->parentNode->replaceChild($newNode, $item);
            }
```